app/domain/authentication/authn_jwt/jwt_configuration_interface.rb

```diff
@@ -4,7 +4,7 @@ module AuthnJwt
     class JWTConfigurationInterface
       def get_identity; end;
```

Comment:
i would call this `extract_conjur_id_from_request` to be more explicit but leaving it up to you

Reply:
There is enchanse logs and supportability issue in our jira

Reply:
Renamed to get_conjur_id